tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/clean_mapping_test.py

```diff
@@ -0,0 +1,184 @@
+        (150, 'chs', 2000)
+""")
+
+OBSERVATION_TABLE_TEMPLATE = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.observation` (
+    observation_id, person_id, observation_concept_id, observation_date, observation_datetime, 
+    observation_type_concept_id, value_as_number, value_as_string, value_as_concept_id, qualifier_concept_id, 
+    unit_concept_id, provider_id, visit_occurrence_id, visit_detail_id, observation_source_value,
+    observation_source_concept_id, unit_source_value, qualifier_source_value, value_source_concept_id, value_source_value, 
+    questionnaire_response_id
+    )
+
+VALUES
+    (100, 1, 1585838, '2009-04-29', TIMESTAMP('2009-04-29'),
+     8201211115, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99051',
+     1585838, NULL, NULL, 1585840, NULL, NULL),
+
+    (200, 1, 1585845, '2009-04-29', TIMESTAMP('2009-04-29'),
+     8201211115, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99051',
+     1585845, NULL, NULL, 1585846, NULL, NULL),
+
+    (300, 2, 1585845, '2009-05-15', TIMESTAMP('2009-05-15'),
+     8201211116, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99054',
+     1585845, NULL, NULL, 1585846, NULL, NULL),
+
+    (400, 3, 1585838, '2009-05-15', TIMESTAMP('2009-05-15'),
+     8201211116, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99054',
+     1585838, NULL, NULL, 1585839, NULL, NULL)
+""")
+
+
+class CleanMappingExtTablesTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    @mock.patch('cdr_cleaner.cleaning_rules.clean_mapping.get_mapping_tables')
+    def setUpClass(cls, mock_mapping_tables):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # Set the test project identifier
+        cls.project_id = os.environ.get(PROJECT_ID)
+
+        # Set the expected test datasets
+        cls.dataset_id = os.environ.get('UNIONED_DATASET_ID')
+        cls.sandbox_id = f'{cls.dataset_id}_sandbox'
+
+        # mock mapping_tables
+        cls.tables = ['_mapping_observation', 'observation_ext']
+        mock_mapping_tables.return_value = cls.tables
+
+        # Instantiate class
+        cls.rule_instance = CleanMappingExtTables(cls.project_id,
+                                                  cls.dataset_id,
+                                                  cls.sandbox_id)
+
+        # Generate sandbox table names
+        sandbox_table_names = cls.rule_instance.get_sandbox_tablenames()
+        for table_name in sandbox_table_names:
+            cls.fq_sandbox_table_names.append(
+                f'{cls.project_id}.{cls.sandbox_id}.{table_name}')
+
+        # Store affected table names
+        affected_tables = cls.rule_instance.affected_tables
+        for table_name in affected_tables:
+            cls.fq_table_names.append(
+                f'{cls.project_id}.{cls.dataset_id}.{table_name}')
```

Comment:
Let's also add the `OBSERVATION` table to `cls.fq_table_names`. `cls.fq_table_names` is supposed to have all the tables this test expects before running the cleaning rule.